content/en/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md

```diff
@@ -0,0 +1,206 @@
+
+A [Kubernetes Ingress Resources](https://kubernetes.io/docs/concepts/services-networking/ingress/) exposes HTTP and HTTPS routes from outside the cluster to services within the cluster.
+
+Let's see how you can configure a `Ingress` on port 80 for HTTP traffic.
+
+1.  Create an Istio `Gateway`:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+      annotations:
+        kubernetes.io/ingress.class: istio
+      name: ingress
+    spec:
+      rules:
+      - host: httpbin.example.com
+        http:
+          paths:
+          - path: /status/*
+            backend:
+              serviceName: httpbin
+              servicePort: 8000
+    EOF
+    {{< /text >}}
+
+    The `kubernetes.io/ingress.class` annotation is required to tell Istio that it should handle this `Ingress`, otherwise it will be ignored.
+
+1.  Access the _httpbin_ service using _curl_:
+
+    {{< text bash >}}
+    $ curl -I -HHost:httpbin.example.com http://$INGRESS_HOST:$INGRESS_PORT/status/200
+    HTTP/1.1 200 OK
+    server: envoy
+    date: Mon, 29 Jan 2018 04:45:49 GMT
+    content-type: text/html; charset=utf-8
+    access-control-allow-origin: *
+    access-control-allow-credentials: true
+    content-length: 0
+    x-envoy-upstream-service-time: 48
+    {{< /text >}}
+
+    Note that you use the `-H` flag to set the _Host_ HTTP header to
+    "httpbin.example.com". This is needed because the `Ingress` is configured to handle "httpbin.example.com",
+    but in your test environment you have no DNS binding for that host and are simply sending your request to the ingress IP.
+
+1.  Access any other URL that has not been explicitly exposed. You should see an HTTP 404 error:
+
+    {{< text bash >}}
+    $ curl -I -HHost:httpbin.example.com http://$INGRESS_HOST:$INGRESS_PORT/headers
+    HTTP/1.1 404 Not Found
+    date: Mon, 29 Jan 2018 04:45:49 GMT
+    server: envoy
+    content-length: 0
+    {{< /text >}}
+
+## Next Steps
+
+### TLS
+
+`Ingress` supports [specifying TLS settings](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls). This is supported by Istio, but the referenced `Secret` must exist in the namespace of the `istio-ingressgateway` deployment (typical `istio-system`). [cert-manager](/docs/ops/integrations/certmanager/) can be used to generate these certificates.
+
+### Specifying path type
+
+By default, Istio will treat paths as exact matches, unless they end in `/*` or `.*`, in which case they will become prefix matches.
+
+In Kubernetes 1.18, a new field, `pathType`, was added. This allows explicitly declaring a path as `Exact` or `Prefix`
+
+### Specifying `IngressClass`
+
+In Kubernetes 1.18, a new object, `IngressClass`, was added, replacing the `kubernetes.io/ingress.class` annotation on the `Ingress` object. If you are using this object, you will need to set the `controller` field to `istio.io/ingress-controller`. For example,
```

Comment:
s/object/resource/

Reply:
I think it would be good to show an example of an Ingress w/ IngressClass. For example:
```suggestion
In Kubernetes 1.18, a new resource, `IngressClass`, was added, replacing the `kubernetes.io/ingress.class` annotation on the `Ingress` resource. For example:

{{< text yaml >}}
apiVersion: networking.k8s.io/v1beta1
kind: Ingress
metadata:
  name: ingress-test
spec:
  ingressClassName: istio-test
  ... 
{{< /text >}}

If you use an `IngressClass` in Istio, the `controller` field must be set to `istio.io/ingress-controller`:
```